### PR TITLE
sparse checkout of GEOS_util

### PR DESCRIPTION
**Pull request by weiyuan-jiang**
This PR needs [GEOS Util PR#58](https://github.com/GEOS-ESM/GEOS_Util/pull/58/files)

### CODE REVIEW

File: components.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ GEOS_Util:
   local: ./src/Shared/@GMAO_Shared/@GEOS_Util
   remote: ../GEOS_Util.git
   tag: v2.0.3
+  sparse: ./config/GEOS_Util.sparse
```
Comment (gmao-rreichle) [inline components.yaml:40]: The config file might need to be named more specifically for GEOSldas, something like `GEOS_Util_ldas.sparse`  (taking my cues from the config file for GEOSgcm_GridComp)

Comment (weiyuan-jiang) [reply components.yaml:40]: This sparse config file belongs to GEOSldas, so "ldas" probably is not necessary

**Comment by gmao-rreichle** [comment]
@weiyuan-jiang : Did you test this PR (using the GEOS_Util feature branch)?  I suppose that would also count as a test of  https://github.com/GEOS-ESM/GEOS_Util/pull/58/files.   I imagine Scott would also need us to test https://github.com/GEOS-ESM/GEOS_Util/pull/58 with the GCM, which @biljanaorescanin could do.   If https://github.com/GEOS-ESM/GEOS_Util/pull/58 is ready, now's a good time to elevate it to "ready for review" and get Scott's attention.  (We also have https://github.com/GEOS-ESM/GEOS_Util/pull/53 waiting for Scott.)

**Comment by weiyuan-jiang** [comment]
> @weiyuan-jiang : Did you test this PR (using the GEOS_Util feature branch)? I suppose that would also count as a test of https://github.com/GEOS-ESM/GEOS_Util/pull/58/files. I imagine Scott would also need us to test [GEOS-ESM/GEOS_Util#58](https://github.com/GEOS-ESM/GEOS_Util/pull/58) with the GCM, which @biljanaorescanin could do. If [GEOS-ESM/GEOS_Util#58](https://github.com/GEOS-ESM/GEOS_Util/pull/58) is ready, now's a good time to elevate it to "ready for review" and get Scott's attention. (We also have [GEOS-ESM/GEOS_Util#53](https://github.com/GEOS-ESM/GEOS_Util/pull/53) waiting for Scott.)

I have built it but not tested

**Comment by gmao-rreichle** [comment]
> I have built it but not tested. I assumed GEOS_Util 's PR is good to go

@weiyuan-jiang: https://github.com/GEOS-ESM/GEOS_Util/pull/58 (**58**) was  started by you and is in draft form, without documentation or notes about testing.  Maybe you confused **58** with https://github.com/GEOS-ESM/GEOS_Util/pull/53 (**53**) when you wrote that you "assumed GEOS_Util 's PR is good to go"?

**Review by gmao-rreichle — APPROVED** [review]

**Pull request merged**